### DISCUSSION TITLE
Sulu Admin – customization with my own React components

### DISCUSSION THREAD

**Discussion by SuperFlySly** [sulu/sulu#7533] [Q&A] [answered]
Hi guys, does anyone know where I might find a tutorial describing how I can extend the admin area with my own React components?

**Comment by alexander-schranz** [comment, marked as answer]
The demo has a view example pull request:

https://github.com/sulu/sulu-demo/pulls

Most important for customizazion is:

https://github.com/sulu/sulu-demo/pull/79

And

https://github.com/sulu/sulu-demo/pull/78

For the react part have look at the tests of the different components and https://jsdocs.sulu.io/2.5/
